### DISCUSSION TITLE
QuickLook lighting workaround?

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#3380] [Q&A]
Hi everyone, I have seen many threads about this same issue but I haven't found any solutions to this problem. When comparing android and iOS models in AR there is a HUGE difference in model color (because of environment lighting from what I understand). Did anyone find a way to make the difference at least less visible? Or is there a way to turn of light estimation / environment lighting in QuickLook?

**Comment by elalish** [comment]
Sadly no, there is very little control over native app rendering. Please ask Apple to support WebXR on iOS so that we can create a consistent experience.

**Reply by danilokrulj** [reply]
Thank you for the reply. I see you are answering almost every question here, keep up the good work!
